pom.xml

```diff
@@ -492,6 +492,7 @@
                         <exclude>**/*.pem</exclude>
 
                         <exclude>.m2/**</exclude>
+                        <exclude>~/.m2/**</exclude>
```

Comment:
Why need this? Seem strange

Reply:
When issuing `./mvnw verify`, I get rat errors as below
```
Files with unapproved licenses:

  ~/.m2/repository/commons-codec/commons-codec/1.9/commons-codec-1.9.jar.sha1
  ~/.m2/repository/commons-codec/commons-codec/1.9/_remote.repositories
  ~/.m2/repository/commons-codec/commons-codec/1.9/commons-codec-1.9.pom.sha1
  ~/.m2/repository/commons-codec/commons-codec/1.6/commons-codec-1.6.pom.sha1
  ~/.m2/repository/commons-codec/commons-codec/1.6/commons-codec-1.6.jar.sha1
  ~/.m2/repository/commons-codec/commons-codec/1.6/_remote.repositories
....
```
Any other suggestions more than this line?

Reply:
Strange, never have jar in this folder

Reply:
The `~/.m2` is not in the scope of the project and must not be analyzed by `./mvnw verify`, unless your local maven settings is wrong that makes it generate a real folder named `~` under the project, if that's the case, `git status` will also show they're untracked, just a hint